index.html

```diff
@@ -65,6 +63,44 @@ <h2>I am a Software Engineer</h2>
     </section>
 
     <section id="works">
+      <!-- popup window for each project -->
+      <div id="popup">
+        <div id="popup-content">
+          <div id="popup-title-X">
+            <h3>Project Title</h3>
+            <button id="popup-X-btn" onclick="hidePopup()">
+              <img src="./imgs/x.svg" alt="">
+            </button>
+          </div>
+
+          <div id="popup-skill-btns">
+            <ul class="nav-buttons">
+              <!-- <li><a href=""></a></li> -->
+              
+            </ul>
+          </div>
+
+          <div id="popup-info">
+            <img width="100%" src="./imgs/works-bg.png" alt="project-image">
+            
+            <div id="popup-info-details">
+              <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent</p>
+              
+              <div id="function-btns">
```

Comment:
- @khanh-devos  from the project requirements you are to render the projects done dynamically with javascript, but you rendered from the HTML. You should have a project array that holds the project details and, your projects rendered from the project array.  This will help you add new projects easily in the future by just adding to the array.

```javascript
const projects = [{
    name: "Submarine game",
    description: "A Phaser3 platform game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
}]
```
- Also Modal popup data is populated with the data from the projects array.